react/components/Map.js

```diff
@@ -159,163 +244,396 @@ class Map extends Component {
       this.recenterMap(this.getLocation(geoCoordinates))
     }
 
-    this.createNewMarkers({
-      pickups: this.props.pickupOptionGeolocations,
-      pickupOptions: this.props.pickupOptions,
-      pickupPoint: this.props.pickupPoint,
-      selectedPickupPointGeolocation: this.props.selectedPickupPointGeolocation,
-      recenter: true,
-      ...(this.props.address.geoCoordinates.value.length > 0
-        ? { address: this.props.address }
-        : {}),
-    })
+    this.createNewMarkers()
+  }
+
+  toggleSearchArea = () => {
+    const newCenter = this.map && this.map.getCenter()
+
+    if (!newCenter || !this.center) return
+
+    const differenceLat = (this.center.lat() - newCenter.lat()) * 1000
+    const differenceLng = (this.center.lng() - newCenter.lng()) * 1000
+
+    const DISTANCE = 50
+
+    const outerLat = differenceLat > DISTANCE || differenceLat < -DISTANCE
```

Comment:
`-DISTANCE` indeed?

Reply:
positive distance and negative distance indeed